Fix for #24104 - fixed infinite loop

dmatej opened the pull request:
- added escape path if it is not possible to find openable free port.
- fixes #24104

dmatej committed:
Fix for #24104 - fixed infinite loop

- added escape path if it is not possible to find openable free port.

Signed-off-by: David Matějček <david.matejcek@omnifish.ee>

pzygielo commented on nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/domain/CustomTokenClient.java:
```diff
@@ -111,6 +112,9 @@ public Map<String, String> getSubstitutableTokens() throws DomainException {
                             // Find next available unused port by incrementing the port value by 1
                             while (!NetUtils.isPortFree(port) && !usedPorts.contains(port)) {
+                                if (port > NetUtils.MAX_PORT) {
+                                    throw new DomainException("No free port available or it is prohibited.");
```
Seems that other `DomainException`s here use `_strings` to get the message.

dmatej replied:
It just extends RepositoryException which extends java.lang.Exception, no i18n inside. The plan is to get rid all of these messages as they are pretty inconsistent and often copy pasted same to a property file and same "as a default". But that will be for another pr(s).

dmatej committed:
Cleanup in i18n related to previous commit and it's PR

- removed unused properties
- moved properties in incorrect files
- deleted one file and it's content was set where appropriate
- properties with the same key on other places were checked too.

Signed-off-by: David Matějček <david.matejcek@omnifish.ee>
